libraries/rush-lib/src/logic/operations/test/AsyncOperationQueue.test.ts

```diff
@@ -37,9 +39,14 @@ describe(AsyncOperationQueue.name, () => {
     const queue: AsyncOperationQueue = new AsyncOperationQueue(operations, nullSort);
     for await (const operation of queue) {
       actualOrder.push(operation);
+      if (operation === UNASSIGNED_OPERATION) {
+        continue;
+      }
```

Comment:
Since nothing in this test sets the `RemoteExecuting` status, this should be an error if it happens.